### PR TITLE
check isPresent before calling get on Optional

### PR DESCRIPTION
**Pull request by holmeso**
# Description

Fixes instances where `Optional.get()` is called without a prior `isPresent()` call.

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Existing unit tests pass

# Are WDL Updates Required?

No wdl updates required

### CODE REVIEW

File: q3vcftools/src/au/edu/qimr/vcftools/util/MergeUtils.java [hunk 1]
```diff
@@ -423,7 +423,7 @@ public static VcfRecord mergeRecords(Map<Integer, Map<String, String>> rules, Vc
 			 */
 			
 			Optional<String> combinedAlts = getCombinedAlt(caller1, caller2);
-			mr = getBaseVcfRecordDetails(caller1, combinedAlts.get());
+			mr = getBaseVcfRecordDetails(caller1, combinedAlts.isPresent() ? combinedAlts.get() : null);
```
Comment (delocalizer) [inline q3vcftools/src/au/edu/qimr/vcftools/util/MergeUtils.java:426]: `combinedAlts.orElse(null)` might be a bit more elegant idiom

File: q3vcftools/src/au/edu/qimr/vcftools/util/MergeUtils.java [hunk 2]
```diff
@@ -530,14 +530,14 @@ public static VcfRecord mergeRecords(Map<Integer, Map<String, String>> rules, Vc
 		 * build up alt string
 		 */
 		
-		Optional<String> combinedAlt = getCombinedAlt(records);
-//		assert combinedAlt.isPresent() : "Null returned from getCombinedAlt";
+		Optional<String> combinedAltO = getCombinedAlt(records);
+		String combinedAlt = combinedAltO.isPresent() ? combinedAltO.get() : null;
```
Comment (delocalizer) [inline q3vcftools/src/au/edu/qimr/vcftools/util/MergeUtils.java:534]: ditto

**Review by delocalizer — APPROVED** [review]
LGTM

**Comment by holmeso** [comment]
agree - `ifElse` is more elegant

**Pull request merged**